solutions/devsprint-haldny-santos-1/app/src/main/java/com/devpass/spaceapp/presentation/launch_list/LaunchListActivity.kt

```diff
@@ -21,6 +25,16 @@ class LaunchListActivity : AppCompatActivity() {
 
     private val viewModel by viewModels<LaunchListViewModel>()
 
+    private val snackBarError by lazy {
```

Comment:
podemos trocar a forma de mostrar o erro? que tal criarmos um AlertDialog, eu acho que fica melhor para o usuário  entender o erro.

Reply:
Sim, abri esse MR com essa alteração: https://github.com/devpass-tech/challenge-mvvm-space-android/compare/main...isabellamartinsc:challenge-mvvm-space-android:feature/add-loading-to-launch-list-activity, ficou melhor? So não sabia muito bem o que escrever rs